driver/src/test/java/org/neo4j/driver/internal/cluster/GetServersProcedureRunnerTest.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2002-2017 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.neo4j.driver.internal.cluster;
+
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.List;
+
+import org.neo4j.driver.internal.net.BoltServerAddress;
+import org.neo4j.driver.internal.spi.Connection;
+import org.neo4j.driver.internal.summary.InternalServerInfo;
+import org.neo4j.driver.v1.Record;
+import org.neo4j.driver.v1.Statement;
+import org.neo4j.driver.v1.Value;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.neo4j.driver.v1.Values.value;
+
+public class GetServersProcedureRunnerTest
+{
+    @Test
+    public void shouldCallGetServersV2WithNull() throws Throwable
+    {
+        // Given
+        GetServersProcedureRunner runner = new TestGetServersProcedureRunner( value( (Object)null ) );
+        Connection mock = mock( Connection.class );
+        when( mock.server() ).thenReturn(
+                new InternalServerInfo( new BoltServerAddress( "123:45" ), "Neo4j/3.2.1" ) );
+        // When
+        runner.run( mock );
+
+        // Then
+        assertThat( runner.procedureCalled().toString(), equalTo(
```

Comment:
`Statement` as a good `#equals()` method defined. I think it should be used for assertions instead of `#toString()`.